distribution/docker/README.md

```diff
@@ -0,0 +1,30 @@
+
+# Getting started with Docker example
+This project is a result of completing [Getting started with Docker](http://knotx.io/tutorials/getting-started-with-docker/2_0/) tutorial.
+
```

Comment:
I would add:
`This example is built using the [Starter Kit](https://github.com/Knotx/knotx-starter-kit) template project.`